lib/fs.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const fs = require('fs');
+
+const mkdir = path => {
+  if (!fs.existsSync(path))
+    fs.mkdirSync(path, err => {
+      if (err) console.error(err);
+    });
+};
+
+const mkdirp = path => {
+  const version = process.versions.node.split('.').map(el => parseInt(el));
+  if (version[0] < 10 || (version[0] === 10 && version[1] <= 11)) {
+    let firstIndex = 0;
+    for (let i = 1; i < path.split('/').length; i++) {
+      mkdir(path.substring(0, path.indexOf('/', firstIndex)));
+      firstIndex = path.indexOf('/', firstIndex) + 1;
+    }
+    mkdir(path);
+  } else {
+    fs.mkdir(path, { recursive: true }, err => {
+      if (err) console.error(err);
```

Comment:
As I mentioned in [comment](https://github.com/metarhia/common/pull/249#discussion_r253792011) error should be passed to callback.